debian/make-preseed-iso.sh

```diff
@@ -6,6 +6,7 @@ set -e
 
 function extract_iso() {
   echo "Extracting iso: $1..."
+  rm -rf isofiles
```

Comment:
can you do a `[ -d isofiles ] && rm -rf isofiles` to quash any unnecessary output if the directory doesn't exist